conans/client/migrations_settings.py

```diff
@@ -367,3 +367,82 @@
 """
 
 settings_1_15_1 = settings_1_15_0
+
```

Comment:
Do not introduce changes in this module. We will do it once a version is released.